docs/release-notes.rst

```diff
@@ -0,0 +1,207 @@
+  (:pull:`59` through :pull:`66`).  By `Pascal Bourgault
+  <https://github.com/aulemahal>`_.
+* Removed internal use of the deprecated :py:class:`numpy.object` data type
+  (:pull:`56` through :pull:`66`).  By `Mathias Hauser
+  <https://github.com/mathause>`_.
+* Updated internals for compatibility with the now calendar-aware
+  :py:class:`cftime.datetime` object (:pull:`51` through :pull:`66`).  By `Jeff
+  Whitaker <https://github.com/jswhit>`_.
+* Improved handling of scalar values passed to
+  :py:meth:`NetCDFTimeConverter.convert` (:issue:`45`, :pull:`46`).  By `Spencer
+  Clark <https://github.com/spencerkclark>`_.
+
+
+v1.2.0 (January 25th, 2019)
+---------------------------
+
+Infrastructure
+~~~~~~~~~~~~~~
+
+* Included license in package data (:pull:`37`).  By `Filipe Fernandes
+  <https://github.com/ocefpaf>`_.
+
+New features
+~~~~~~~~~~~~
+
+* Added ability to directly plot subclasses of :py:class:`cftime.datetime`, e.g.
+  :py:class:`cftime.DatetimeNoLeap`, instead of requiring
+  :py:class:`CalendarDateTime` objects (:pull:`42`).  By `Spencer
+  Clark <https://github.com/spencerkclark>`_.
+
+
+v1.1.0 (May 31st, 2018)
+-----------------------
+
+Requirements
+~~~~~~~~~~~~
+
+* nc-time-axis now requires matplotlib of at least version 2.0 (:issue:`23`,
+  :pull:`34`).  By `Filipe Fernandes <https://github.com/ocefpaf>`_.
+* nc-time-axis now uses the standalone cftime package instead of the
+  ``netcdftime`` module formerly packaged in netcdf4-python (:pull:`30`,
+  :pull:`32`). By `Filipe Fernandes <https://github.com/ocefpaf>`_.
+
+Infrastructure
+~~~~~~~~~~~~~~
+
+* Updated continuous integration tests to be run with both Python 2 and Python 3
+  (:pull:`33`).  By `Filipe Fernandes <https://github.com/ocefpaf>`_.
+* Updated continuous integration to use ``install_requires`` information to
+  install dependencies of nc-time-axis instead of a requirements file
+  (:issue:`27`, :pull:`28`).  By `Phil Elson <https://github.com/pelson>`_.
+* Added installation and test requirements to ``setup.py`` (:pull:`26`).  By
+  `Luke Carroll <https://github.com/LukeC92>`_.
+
+Documentation
+~~~~~~~~~~~~~
+
+* Added installation instructions and a usage example to the README for display
+  on GitHub (:pull:`24`, :pull:`25`).  By `mbeedie
+  <https://github.com/mbeedie>`_.
+
+
+v1.0.2 (March 7th, 2017)
+------------------------
+
+Requirements
+~~~~~~~~~~~~
+
+* nc-time-axis now requires matplotlib less than version 2.0 (:pull:`22`).  By
+  `Mark Hedley <https://github.com/marqh>`_.
+
+Bug fixes
+~~~~~~~~~
+
+* Fixed a bug in comparing calendars (:pull:`22`).  By `Mark
+  Hedley <https://github.com/marqh>`_.
+
+
+v1.0.1 (November 23rd, 2016)
+----------------------------
+
+Bug fixes
+~~~~~~~~~
+
+* Fixed a bug the prevented converting NumPy arrays of datetime objects.  This
+  enables making Hovmoller diagrams using nc-time-axis.  By `Peter Killick
+  <https://github.com/DPeterK>`_.
+
+
+v1.0.0 (July 1st, 2016)
```

Comment:
This history looks about right. As I recal, it was a prototype from me, and a productionisation from lbdreyer, so think you have the balance of credit about right 👍

Reply:
Great thanks!  I added a short sentence to this effect in the top-level summary of the initial release in #89, which hopefully makes these contributions by both you and @lbdreyer a little more prominent.